addons/fleet/views/fleet_vehicle_cost_views.xml

```diff
@@ -175,6 +175,9 @@
                     <button name="contract_close" states="open,expired,future" type="object" class="oe_highlight" string="Close Contract" groups="fleet.fleet_group_manager"/>
                     <button name="contract_open" states="closed" type="object" class="oe_highlight" string="Set Contract In Progress" groups="fleet.fleet_group_manager"/>
                     <button class="oe_highlight" name="act_renew_contract" type="object" string="Renew Contract" help="Create a new contract automatically with all the same informations except for the date that will start at the end of current contract" groups="fleet.fleet_group_manager"/>
+                    <button name="run_scheduler" type="object"
```

Comment:
It is necessary ? Why ? I want to know. Can't sleep sir ! plz plz

Reply:
I added it to run the scheduler quickly. I can remove it if you want.